src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/main.js

```diff
@@ -90,6 +100,16 @@ define([
             },
 
             /**
+             * raised after a media got moved
+             * @event sulu.media.collections.media-moved
+             * @param id {Number|String} the id of the moved media
+             * @param collection {Object} collection moved media into
+             */
+            SINGLE_MEDIA_MOVED = function() {
```

Comment:
this event is not used currently (because a callback is passed) and i have used the mechanism of delete ...